docs/source/first-steps.rst

```diff
@@ -20,6 +20,20 @@ using your tCRIL email address. Verify your email address using the link sent
 from Jira. Use the grid of nine dots in the upper left corner to switch between
 Jira and Confluence.
 
+You'll also need to get yourself added to the ``TCRIL`` Atlassian Group.  `File
```

Comment:
Since these instructions are meant for all 'krillers, does it make sense to have this under a "for engineers" section? or should we give all krillers this access?

Reply:
I think community managers and other people working with the Open edX Platform should also get this level of access.  Though we can refine this if we find that it doesn't make sense for everyone in the future.